.circleci/config.yml

```diff
@@ -23,7 +23,61 @@ jobs:
             brew install shellcheck
             bundle check || bundle install --jobs=4 --retry=3 --path .bundle
       - save_cache:
-          key: v1-gems-{{ checksum "Gemfile" }}
+          key: fastlane-macos-gems-{{ checksum "Gemfile" }}
+          paths:
+            - .bundle
+
+      - run:
+          name: Check PR Metadata
+          command: bundle exec danger || echo "danger failed"
+
+      - run: bundle exec fastlane test
+
+      - store_test_results:
+          path: /Users/distiller/test-results
+
+      - run:
+          name: Post Test Results to GitHub
+          command: bundle exec danger || echo "danger failed"
+          when: always # Run this even when tests fail
+  
+  "Ubuntu":
+    environment:
+      CIRCLE_TEST_RESULTS: $HOME/test-results
+      LC_ALL: C.UTF-8
+      LANG: C.UTF-8
+      FASTLANE_ITUNES_TRANSPORTER_PATH: .bundle
+    docker:
+      - image: circleci/ruby:2.3
+    shell: /bin/bash --login -eo pipefail
```

Comment:
Probably don't need a login shell on linux?

Reply:
Will look into what this actually does/means. Stupid copy/paste from macOS for now.

Reply:
(Why do we need this on macOS?)

Reply:
Read up on it:
Don't think we need that.

https://circleci.com/docs/2.0/configuration-reference/#default-shell-options


(Why do we need this on macOS?)

Reply:
Read up on it:
Don't think we need that.

https://circleci.com/docs/2.0/configuration-reference/#default-shell-options

(Why do we need this on macOS?)

Reply:
Read up on it: https://www.gnu.org/software/bash/manual/html_node/Bash-Startup-Files.html#Bash-Startup-Files
Don't think we need that.

(Why do we need this on macOS?)

When `--login` is gone, all that is left is actually the default: https://circleci.com/docs/2.0/configuration-reference/#default-shell-options
So this line can be completely removed 👍

Reply:
We need it on macOS jobs so that we source the `chruby` init scripts, to use our chosen ruby, and not system ruby